templates/index.html

```diff
@@ -0,0 +1,205 @@
+                            <input type="text" class="terms" name="parameter" placeholder="Enter your parameter here:"
+                                   required>
+                        </div>
+                        <input type="submit" name="submit" value="Search Terms In Our Database"
+                               style="background-color: black;">
+                    </form>
+                </section>
+            </div>
+        </div>
+    </section>
+
+    <!--Search in User Database-->
+    <section id="user_database" class="wrapper user_database fade-up">
+        <div class="inner">
+            <center><h1><i>User Database Search</i></h1></center>
+            <div class="features">
+                <section>
+                    <h2>Paste your MEI and upload files</h2>
+                    <div class="w3-row-padding" style="margin:0 -2px;">
+                        <div class="w3-full">
+                            <form method="POST" enctype="multipart/form-data">
+                                <textarea name="text" rows="9" cols="80" required></textarea>
+                                <br>
+                                <input type="file" name="base_file" multiple="" required>
+                                <input type="submit" name="submit" value="Upload and Search Your Snippet">
+                            </form>
+                        </div>
+                    </div>
+                </section>
+                <section>
+                    <h2>Upload files and input parameters to search</h2>
+                    <form method="POST" enctype="multipart/form-data">
+                        <div style="padding: 20px 0 40px 0;">
+                            <select name="term">
+                                <option value="">Select your term</option>
+                                <option value="Expressive Terms">Expressive Terms</option>
+                                <option value="Tempo Marking">Tempo Marking</option>
+                                <option value="Articulation">Articulation</option>
+                                <option value="Dynamic Markings">Dynamic Markings</option>
+                                <option value="Piano Fingerings">Piano Fingerings</option>
+                                <option value="Pedal Marking">Pedal Marking</option>
+                                <option value="Hairpin">Hairpin</option>
+                            </select>
+                            <input type="text" class="terms" name="parameter" placeholder="Enter your parameter here:"
+                                   required>
+                        </div>
+                        <input type="file" name="base_file" multiple="" required>
+                        <input type="submit" name="submit" value="Upload and Search Parameter">
+                    </form>
+                </section>
+            </div>
+        </div>
+    </section>
+
+    <!-- Two -->

```

Comment:
What's this stuff here?

Reply:
Ughhh that was the placeholder I typed, will delete.

Reply:
@yinbogao @Lijia-Li delete?

Reply:
done! I think we will add in introduction and user documentation to the page at some point tonight.